core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -1590,8 +1514,8 @@ export class ConversationSkinComponent {
       },
       0.1 * TIME_FADEOUT_MSEC +
         TIME_HEIGHT_CHANGE_MSEC +
-        TIME_FADEIN_MSEC +
-        this.TIME_PADDING_MSEC
+        this.conversationFlowService.TIME_FADEIN_MSEC +
```

Comment:
I think that the fadein times should be properties of the controller/component, not the service? At the very least the fadeout and fadein times should be in the same file.

Reply:
Done